adapters/repos/db/vector_index.go

```diff
@@ -21,6 +21,7 @@ import (
 // VectorIndex is anything that indexes vectors efficiently. For an example
 // look at ./vector/hsnw/index.go
 type VectorIndex interface {
+	Dump(labels ...string)
```

Comment:
Is this method implemented?

Reply:
Visual studio isn't showing the implementations, but it is defined.  I don't think go will compile successfully unless all the functions are implemented.